morl_baselines/common/utils.py

```diff
@@ -63,3 +65,56 @@ def make_gif(env, agent, weight: np.ndarray, fullpath: str, fps: int = 50, lengt
     clip = ImageSequenceClip(list(frames), fps=fps)
     clip.write_gif(fullpath + ".gif", fps=fps)
     print("Saved gif at: " + fullpath + ".gif")
+
+
+class MaxAndSkipObservationV0(gym.Wrapper):
```

Comment:
This should be moved to MO-Gymnasium

Reply:
I added to Gymnasium (https://github.com/Farama-Foundation/Gymnasium/releases/tag/v0.29.0). But then we need to require >=0.29.0

Reply:
Awesome! I don't have anything against bumping the version in that case.

Reply:
Damn, the one in Gymnasium does not work with vector rewards. I will have to move this one to MO-Gymnasium.